esmvaltool/interface_scripts/variable_info.py

```diff
@@ -380,6 +378,13 @@ def _load_table(self, table_file):
                 if not self._read_line():
                     return
 
+    def add_custom_table_file(self, table_file, table_name):
```

Comment:
What do you think about having this method in a base class that both CMIP5info and CMIP6info classes derive from? I can do the refactor if you want

And one more question, do we keep the old format or go for CMIP6 json? What do you think @mattiarighi?

Reply:
Good question... would it be feasible to maintain both formats? Or is it too much work?
Otherwise I think we should go for CMIP6.

Reply:
@jvegabsc: Having an abstract base class for CMIP?Info is maybe a bit of an overkill with only two child classes with very few public methods, but it's up to you.